en/engine/renderable/model-component.md

```diff
@@ -33,14 +33,13 @@ After batching, the original transform of `MeshRenderer` cannot be changed, but
 
 The engine currently provides two sets of dynamic batching systems, *instancing batching* and *VB-merging batching*. The two methods cannot coexist, and the priority of instancing is greater than that of dynamic merging VB. To enable batching, simply check the `USE_INSTANCING` or `USE_BATCHING` switch in the material used in the model.
```

Comment:
\*instancing batching\* and \*VB-merging batching\*
->
\*\*instancing batching\*\* and \*\*VB-merging batching\*\*